### PR TITLE
use more pythonic syntax

### PR DESCRIPTION
**Pull request by imba-tjd**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright © 2017-present, [Encode OSS Ltd](http://www.encode.io/).
+Copyright © 2017-present, [Encode OSS Ltd](https://www.encode.io/).
```
Comment (Kludex) [inline LICENSE.md]: I think there's no problem on updating this in the license, but just to be sure... Do you know if this is a problem @tomchristie ?

File: uvicorn/server.py [hunk 2]
```diff
@@ -217,7 +217,7 @@ async def main_loop(self) -> None:
         should_exit = await self.on_tick(counter)
         while not should_exit:
             counter += 1
-            counter = counter % 864000
+            counter %= 864000
```
Comment (Kludex) [inline uvicorn/server.py]: Can we revert this one? 🙏 

I always think that the below is previous is easier to read.
```suggestion
            counter = counter % 864000
```

**Review by Kludex — CHANGES_REQUESTED** [review]
Thanks @imba-tjd 🙏 

Can you check my comment below?

**Comment by imba-tjd** [comment]
Test fails 🤔 but the first commit passed.

**Comment by Kludex** [comment]
I've rerun the pipeline. :)

File: LICENSE.md (hunk 1 above)
Comment (lovelydinosaur) [reply LICENSE.md]: It's fiiiiiiiiiiiiiiiinnnnnneeeeeeeeeeeeeeee

Comment (lovelydinosaur) [reply LICENSE.md]: Tho unrelated to the rest of the PR. Which. Hrm.

Comment (lovelydinosaur) [reply LICENSE.md]: If we update that link, let's have it as a separate PR, from someone on the team.

**Review by Kludex — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks @imba-tjd 🙇

**Pull request merged**